.github/workflows/wheel.yml

```diff
@@ -10,21 +13,17 @@ permissions:
 
 on:
   push:
-    # Workflow run on tags for v2.6 and v3 only.
+    # Workflow run on tags for v3 only.
     tags:
-      - v2.6.*
       - v3.*
   pull_request:
     # Workflow run on pull_request only when related files change.
-    branches-ignore:
-      - dev-*
-      - lg-*
-      - rtd
-      - sonar-1
     paths:
       - .github/workflows/wheel.yml
       - pyproject.toml
       - src/python/OpenImageIO/*.py
+      - src/python/OpenImageIO/*.h
+      - src/python/OpenImageIO/*.cpp
```

Comment:
Sorry for not spotting this earlier, but I think all of these should be `src/python/*.h`, .cpp, and .py. There is no OpenImageIO subdirectory in that path.

Reply:
ha -- good point, adjusting now